openupgrade_scripts/scripts/event/16.0.1.7/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,36 @@
+---Models in module 'event'---
+---Fields in module 'event'---
+event        / event.event              / seats_available (integer)     : not stored anymore
+event        / event.event              / seats_reserved (integer)      : not stored anymore
+event        / event.event              / seats_unconfirmed (integer)   : not stored anymore
+event        / event.event              / seats_used (integer)          : not stored anymore
+event        / event.event.ticket       / seats_available (integer)     : not stored anymore
+event        / event.event.ticket       / seats_reserved (integer)      : not stored anymore
+event        / event.event.ticket       / seats_unconfirmed (integer)   : not stored anymore
+event        / event.event.ticket       / seats_used (integer)          : not stored anymore
+# Post-migration: remove the columns in event_event and event_event_ticket tables as fields were and are still readonly
```

Comment:
Here in OpenUpgrade project we don't delete obsolete columns. If you want to delete obsolete columns, please use a database cleanup tool like the module database_cleanup of OCA/server-tools

Reply:
Right, changed